src-opam/distro.ml

```diff
@@ -562,6 +571,7 @@ let distro_status (d : t) : status =
         `Deprecated
     | `Cygwin v -> win10_docker_status `ServerCore v
     | `Windows (_, v) -> win10_docker_status `Windows v
+    | `Macos _ -> `Active `Tier3 (** TODO What does this Tier level mean? Matches Windows Tier3 *)
```

Comment:
Tier level is used to get a grouping on a set of distributions so they can have different properties:

* Tier 1 will have all the different variants and architectures and should be tested thoroughly
* Tier 2 are less supported but still important distributions to test as extra
* Tier 3 are new or badly supported distributions in opam-repository. Those are usually disabled in ocaml-ci (otherwise it would fail too often) and enabled in opam-repo-ci for example (to ensure its support can get better over time)

Reply:
The plan is to support macOS Monterey on x86_64 with a small range of OCaml versions (4.14 and 5.0). That sounds like a Tier2 to me?

In future we would look at supporting Apple Silicon as that is clearly the future for macOS developer tools, but that is planned to come after macOS x86_64 and Windows support is rolled out. We will get more benefit for the effort from hitting macOS and Windows, over an additional architecture that is partially tested on Linux ARM64.

Reply:
Either Tier2 or Tier3 is fine for me (the macos support in opam-repo-ci isn't flowless yet)